plugin.rb

```diff
@@ -19,12 +19,16 @@ module ::DiscourseCustomTopicLists
 
 after_initialize do
   add_to_serializer(:current_user, :custom_topic_lists) do
-    custom_lists = JSON.parse(SiteSetting.custom_topic_lists) || []
-    current_user = scope.user
-    custom_lists.select do |list|
-      allowed_groups = list["access"].split(/(?:,|\s)\s*/).map(&:to_i)
-      allowed_groups = [Group::AUTO_GROUPS[:everyone]] if allowed_groups.empty?
-      current_user.in_any_groups?(allowed_groups)
+    begin
+      custom_lists = JSON.parse(SiteSetting.custom_topic_lists) || []
+      current_user = scope.user
+      custom_lists.select do |list|
+        allowed_groups = list["access"].split(/(?:,|\s)\s*/).map(&:to_i)
+        allowed_groups = [Group::AUTO_GROUPS[:everyone]] if allowed_groups.empty?
+        current_user.in_any_groups?(allowed_groups)
+      end
+    rescue StandardError
```

Comment:
What error do we anticipate here specifically?

Reply:
`JSON::ParserError,` I also just took another, and I think there was a better way to do this...